packages/xchain-cosmos/src/client.ts

```diff
@@ -28,15 +30,24 @@ export interface CosmosClient {
   getMainAsset(): Asset
 }
 
+const MAINNET_SDK = new CosmosSDKClient({
+  server: 'https://api.cosmos.network',
+  chainId: 'cosmoshub-3',
+})
+const TESTNET_SDK = new CosmosSDKClient({
+  server: 'http://lcd.gaia.bigdipper.live:1317',
+  chainId: 'gaia-3a',
+})
```

Comment:
Make it lazy by providing a function, which returns CosmosSDKClient or create it in constructor of Client only. We don't need two instances of `CosmosSDKClient`, just one for testnet OR mainnet.

Reply:
agree, but same as xchain-ethereum, this was done to speed up performance when switching between testnet and mainnet.  i tried to remove as much state as possible at first. i re-added this bit of "caching" back to avoid some expensive setup calls.....